zkvm/src/ops.rs

```diff
@@ -185,7 +181,7 @@ impl Encodable for Instruction {
             Instruction::Program(progitem) => 1 + 4 + progitem.serialized_length(),
             Instruction::Dup(_) => 1 + 4,
             Instruction::Roll(_) => 1 + 4,
-            Instruction::Range(_) => 1 + 1,
+            Instruction::Range => 1 + 1,
```

Comment:
since `range` does not have an argument anymore, the instruction takes 1 byte, not 2.

```suggestion
            Instruction::Range => 1,
```